src/pages/players/players.ts

```diff
@@ -269,6 +270,38 @@ export class PlayersPage {
 		return this.appStorage.getGroups();
 	}
 
+	/**
+	 * Deletes a player from the view and database, after confirmation from user.
+	 *
+	 * @param {Player} player Player to delete.
+	 * @memberof PlayersPage
+	 */
+	private deletePlayer(player: Player): void {
+
+		const confirm = this.alertController.create({
+			title: 'Are you SURE?',
+			message: 'This action cannot be undone. Are you sure you want to delete ' + player.firstName + ' ' +
+					player.lastName + ' from all existence?',
+
+		  buttons: [
```

Comment:
Indentation and spacing here is wonk. I think everything below this should be indented once more. I would remove the empty newlines as well.